irohad/ordering/impl/ordering_service_transport_grpc.cpp

```diff
@@ -38,13 +36,10 @@ grpc::Status OrderingServiceTransportGrpc::onBatch(
           return std::make_shared<shared_model::proto::Transaction>(tx);
         });
 
-    auto batch_result = shared_model::interface::TransactionBatchFactory::
-        createTransactionBatch(
-            txs,
-            shared_model::validation::DefaultSignedTransactionsValidator());
+    auto batch_result = batch_factory_->createTransactionBatch(txs);
```

Comment:
The transactions aren't checked for stateless validity here but it probably should since it grpc endpoint